paddle/scripts/docker/build.sh

```diff
@@ -198,7 +198,7 @@ EOF
     # run paddle version to install python packages first
     RUN apt-get update &&\
         ${NCCL_DEPS}\
-        apt-get install -y wget python-pip dmidecode python-tk && pip install -U pip==9.0.3 && \
+        apt-get install -y wget python-pip dmidecode python-tk && easy_install -U pip && \
```

Comment:
Also need to update `paddle/scripts/paddle_build.sh`, the CI currently use this script now.

Reply:
Thanks, done.